Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.0)
+    oj (3.10.5)
     orm_adapter (0.5.0)
     parallel (1.19.1)
     parser (2.6.5.0)
       ast (~> 2.4.0)
     passenger (6.0.4)
       rack
       rake (>= 0.8.1)
-    polyamorous (2.3.0)
-      activerecord (>= 5.0)
+    polyamorous (2.3.2)
```

Comment:
### [polyamorous](https://github.com/activerecord-hackery/ransack/tree/master/polyamorous)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.3.0 → 2.3.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/activerecord-hackery/ransack/compare/v2.3.0...v2.3.2).

These are the first 10 commits:
- (f2ec1f9) [Remove note about Rails 6.0 support [ci skip]](https://github.com/activerecord-hackery/ransack/commit/f2ec1f935d71dedc1fcfc576fb3bfdbd93f4e03d)
- (92ee7b7) [Fix incompatibility with Dependabot](https://github.com/activerecord-hackery/ransack/commit/92ee7b70ef32b41c72acc9543bce60127261a2b8)
- (f1ac639) [Merge pull request #1057 from y-yagi/remove_note_about_rails…](https://github.com/activerecord-hackery/ransack/commit/f1ac639952dcd1829f67dade6726e5b1f064d031)
- (d3dbed5) [Merge pull request #1058 from nguyenductung/develop](https://github.com/activerecord-hackery/ransack/commit/d3dbed56b94f7894d72dbf110486cf01bda2488a)
- (8c92a3b) [Move development dependencies to Gemfile](https://github.com/activerecord-hackery/ransack/commit/8c92a3b1f6f2360599732f8d1ff4ae95f515eb86)
- (1f5be02) [Fix duplicated mysql2 development dependency](https://github.com/activerecord-hackery/ransack/commit/1f5be0250785be0d3ad8104b615ae5b92241c292)
- (9538bd4) [Make sure the fallback is used](https://github.com/activerecord-hackery/ransack/commit/9538bd4c30f02ad64858616cff9848d5ec1267d8)
- (20d53be) [Bump pg dependency](https://github.com/activerecord-hackery/ransack/commit/20d53be338d2c7f271127fd1905d4ac9bf4d1c7a)
- (96c1fa3) [Remove windows specific dependency](https://github.com/activerecord-hackery/ransack/commit/96c1fa3d0d7863e899b78a8771c8b6a163fb8184)
- (3666807) [Test against the latest ruby](https://github.com/activerecord-hackery/ransack/commit/3666807cd0de7d6013c1c3824ad1b393414f29d7)


</details>